flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/table/SelectITCase.scala

```diff
@@ -134,4 +134,22 @@ class SelectITCase(
       .select('a, 'b as 'a).toDataSet[Row].print()
   }
 
+  @Test
+  def testLiteralOnLeft(): Unit = {
```

Comment:
We want to reduce long running ITCases. We recently introduced an `ExpressionTestBase` for tests of this type. You can create an `ArithmeticExpressionsTest` similar to `DecimalTypeTest`.